notebooks/6_search_in_continuous_spaces/index.md

```diff
@@ -0,0 +1,282 @@
+## Constrained optimization and Unconstrained optimization 
+**Constrained optimization problems** consider the problem of optimizing an objective function subject to constraints on the variables. In general terms,
+
+minimize fx
+
+subject to    cix=0    ∀i ϵ ε
+
+cix≤0    ∀i ϵ τ 
+
+Where *f and* the functions ci(x) are all smooth, real-valued functions on a subset of Rn and ε and τ are index sets for equality and inequality constraints, respectively. The feasible set is the set of points x that satisfy the constraints.
+
+We denote the set of points for which all the constraints are satisfied as C, and say that any x ∈ C (resp. x ∈/ C) is feasible (resp. infeasible)
+
+***Unconstrained optimization problems*** the answers are constrained into being subject of set C as the picture bellow shows: 
+
+![](Aspose.Words.f2571b5b-5e5e-42a8-b357-02f03b593c59.001.png)
+
+
+
+
+
+*Figure  SEQ Figure \\* ARABIC 1. constrained vs unconstrained optimization*
+![](Aspose.Words.f2571b5b-5e5e-42a8-b357-02f03b593c59.002.png)
+
+## Differentiable optimization and Non-differentiable optimization
+*Figure  SEQ Figure \\* ARABIC 2. Non-differentiable function*
+![](Aspose.Words.f2571b5b-5e5e-42a8-b357-02f03b593c59.003.png)Non-differentiable optimization is a category of optimization that deals with objective that for a variety of reasons is non-differentiable and thus non-convex. The functions in this class of optimization are generally non-smooth. These functions although continuous often contain sharp points or corners that do not allow for the solution of a tangent and are thus non-differentiable. In practice non-differentiable optimization encompasses a large variety of problems and a single one-size fits all solution is not applicable however solution is often reached through implementation of the sub gradient method. Non-differentiable functions often arise in real world applications and commonly in the field of economics where cost functions often include sharp points.
+
+![](Aspose.Words.f2571b5b-5e5e-42a8-b357-02f03b593c59.004.png)
+### Cost functions
+In many cases, particularly economics the cost function which is the objective function of an optimization problem is non-differentiable. These non-smooth cost functions may include discontinuities and discontinuous gradients and are often seen in discontinuous physical processes. Optimal solution of these cost functions is a matter of importance to economists but presents a variety of issues when using numerical methods thus leading to the need for special solution methods.
+
+In this lecture we don’t discuss non-differential optimization and non-smooth functions and the text above was for introduction and further information on this topic.

```

Comment:
Try to reformat this sentence and provide at least a reference for further information.